### ISSUE TITLE
LOID/PLOAM passwod

### ISSUE THREAD

**Issue by CharlieDeltaOscar** [yaronkle/ZTE-F601-knowledgebase#1]
Hi, do you know how to get LOID/PLOAM password if I have login/password for web UI and telnet? Thank you!

**Comment by yaronkle** [comment]
@CharlieDeltaOskar 
You can access the LOID (GPON SN) and password from the Web UI when logged in as admin.
Network -> PON
![zte_network_pon](https://github.com/user-attachments/assets/80f6d179-2d3b-4262-afd7-5aa12c1e26ae)

From what I know, the ISP only needs your PON serial number for activation. They do not need the password. Note that in some countries the ISP require that the PON serial number start with a prefix such as ZTEG. The serial number is also on a sticker at the back of the device.

You can also access this via telnet:
`# sendcmd 1 DB p GPONCFG`

I am not familiar with PLAOM password however I see that there are 2 passwords that can be configured via terminal:

`# setmac
Usage format:
    setmac <action type> [<para_id>] [<hex string para_val>]
    setmac show
action type:
    1-set param; (exam: setmac 1 512 sn:20081106)
    2-get param; (exam: setmac 2 512)
    3-del param; (exam: setmac 3 512)
    4-format region; (exam: setmac 4)

# setmac show`
 
You will want to check out 
 `PONPASSWD[ID:  2181] is set to ...
 GPONPWD[ID:  2178] is set to ...`
  
If you change values, a factory reset will not restore these values so you.
Save the values aside before making modifications!

**Comment by CharlieDeltaOscar** [comment]
Thank you for your answers! Unfortunately, the password is hidden with asterisks on my UI, but commands for the terminal should be handy. Also, for some reason, almost all commands start responding with `/bin/sh: Access Denied. ` and I can do nothing about it. I can't even do config backup because tftp is "Access Denied" too.

**Comment by yaronkle** [comment]
Did the shell work before?
Try a reboot. Would hold off with factory reset for now.

Do you have a ZTE-F601 or another device?
What is your Software version?
Did the shell commands work before?

How are you connecting?
telnet 192.168.1.1, username: root, password: Zte521?

**Comment by CharlieDeltaOscar** [comment]
Yes, the shell worked before and I can't figure out what happened. Reboot does not help. 
Here is my device info:
```
Model	F601
Serial Number	XXXXXX-ZTEKXXXXXXXXXX
Hardware Version	V7.0
Software Version	V7.0.10P1T17
Batch Number	07e4P1T170072
Boot Loader Version	V7.0.10T6
```

**Comment by yaronkle** [comment]
I haven't encountered this behavior.

Are you logged in as root?
Can you factory reset the device or is this your working device?
Did you get the ZTE from your ISP? They may have reconfigured it remotely disabling access, although I doubt it.

**Comment by CharlieDeltaOscar** [comment]
Yes, logged as root. I can't reset because it is preconfigured by ISP. I have ordered an identical device and hope I can transfer ISP configuration to it but I don't know how it can be done.

**Comment by yaronkle** [comment]
What would you like to achieve?

Regarding the access denied, not sure I can help much but just in case, can you paste the results of the following:

`sendcmd 61 telnetd ShowCfg`
If you get an "admin" username, try to telnet with username admin and the password that is shown there

If that does not work, paste the results of the following:

`cat /proc/self/status`
`ls -l /bin/`
`ls -l /sbin/`
`getenforce`
`help`
